agent/functional_tests/testdata/taskdefinitions/container-metadata-file-validator-windows/task-definition.json

```diff
@@ -0,0 +1,27 @@
+{
+    "family": "ecsftest-container-metadata-file-validator-windows",
+    "containerDefinitions": [{
+      "image": "amazon/amazon-ecs-container-metadata-file-validator-windows",
+      "name": "container-metadata-file-validator",
+      "memory": 512,
+      "cpu": 1024,
+      "healthCheck": {
```

Comment:
Do we need health check for this test?

Reply:
Other tests that run a custom image use the healthCheck (like v3-task-endpoint-validator). Since this test runs for ~15 seconds, doesn't hurt to have a healthCheck every 5 secs.